.ruby-gemset

```diff
@@ -0,0 +1 @@
+heartcheck
```

Comment:
- I think that is not in the same context of this PR.
- Disable just for my machine will work just for me.  Everybody will need to disable it.

For me we need to change the culture to everybody stop to use gemsets and with that approach you will force everybody to use it our configure to ignore it.

WDYT?